coralproject/talk
# [next] Adding a Message component

## Message Component

### What does this PR do?
- Adds Message Component, needed for Editing
- `ValidationMessage` relies on `Message` Component
<img width="889" alt="screen shot 2018-09-07 at 10 32 42 am" src="https://user-images.githubusercontent.com/1401559/45221926-5fa96200-b289-11e8-9d86-9e031ef564a4.png">

### How do I test this PR?
- run docz
- check the Message Component
- check the values to match the styles, if it does we should correct the ones in `ValidationMessage`

Note: This component is similar to `ValidationMessage`, we could have a `<Message variant="validation" />` or `<Message variant="info" />`, etc.
I like them both separated because it looks semantic. But the API it's up for discussion.  :)

Note2: I know we talked about having `Icon` off this but this has specific styling that's why I think it should be inside the component
